docs/builder.md

```diff
@@ -220,14 +228,16 @@ function truthyValidation(value) {
 }
 
 const field = new BaseBuilder()
-    .setValidation(booleanValidation)
-    .addValidation(truthyValidation);
+    .setValidation(booleanValidation, 'BooleanValidation')
```

Comment:
What is the difference between `setValidation` and `addValidation`?

Reply:
`add` creates a composite validation function. `set` replaces the existing validation with whatever you set it to.